Source/SVGKDefine.h

```diff
@@ -0,0 +1,62 @@
+//
+//  SVGKDefine.h
+//  SVGKit-iOS
+//
+//  Created by lizhuoli on 2018/10/14.
+//  Copyright © 2018年 na. All rights reserved.
```

Comment:
I'd like a sentence or several here explaining why this header exists.

I think it's "a central place to hold lots of project-wide defines that make it easier to write clean code that works across the different Apple platforms, which are sometimes incompatible with each other" ... or something like that! ... but I'm not sure.

Reply:
Done.